source/core/parameters_lagrangian.cc

```diff
@@ -296,28 +297,42 @@ namespace Parameters
                           "1",
                           Patterns::Integer(),
                           "Insertion frequency");
-        prm.declare_entry("insertion direction sequence",
-                          "0,1,2",
-                          Patterns::List(Patterns::Integer(), 2, 3),
-                          "First direction of particle insertion");
+
+        // Clear and Add :
         prm.declare_entry(
-          "insertion box points coordinates",
+          "list of input files",
+          "particles.input",
+          Patterns::List(Patterns::FileName()),
+          "List of files from which we load the particles using the clear_and_add method");
+        prm.declare_entry(
+          "clearing box points coordinates",
           "0. , 0. , 0. : 1. , 1. , 1.",
           Patterns::List(
             Patterns::List(Patterns::Double(), 2, 3, ","), 2, 2, ":"),
-          "Coordinates of two points (x1, y1, z1 : x2, y2, z2)");
-        prm.declare_entry("insertion distance threshold",
-                          "1",
-                          Patterns::Double(),
-                          "Distance threshold");
-        prm.declare_entry("insertion maximum offset",
-                          "1",
-                          Patterns::Double(),
-                          "Maximum position offset went insertion particles");
-        prm.declare_entry("insertion prn seed",
-                          "1",
-                          Patterns::Integer(),
-                          "Prn seed used to generate the position offsets");
+          "Coordinates of two points for the clearing box (x1, y1, z1 : x2, y2, z2)");
```

Comment:
Why do you need a new list of filenames?
I would use the previous list of filenames for the file insertion mechanism.